operations/list.go

```diff
@@ -236,7 +237,9 @@ func listAliases(ctx context.Context, confPath, project, filename string) error
 	}
 
 	for _, alias := range aliases {
-		fmt.Printf("%s\t%s\t%s\n", alias.Alias, alias.Variant, alias.Task)
+		if alias.Alias != "__github" {
```

Comment:
Use the "__github" constant from the `model` package.

Reply:
I found the "__github" constant in the patch package and assumed that was what you were referring to. Was that correct?

Reply:
Yes.